src/storage/src/mem_table.rs

```diff
@@ -81,18 +84,22 @@ impl MemTable {
     /// write methods
     pub fn insert(&mut self, pk: Bytes, value: Bytes) -> Result<()> {
         if !self.is_consistent_op {
+            self.mem_table_size += std::mem::size_of::<Bytes>() + pk.len() + value.len();
             self.buffer.insert(pk, KeyOp::Insert(value));
```

Comment:
when `!is_consistent_op`, an insert may overwrite the original value, and therefore we should decrease the size of original value.

Reply:
Question: When or which executors' `is_consistent_op` is false?